w3-aggregation.md

```diff
@@ -127,20 +127,23 @@ type AggregateOffer struct {
 
 type AggregateOfferDetail struct {
   offer OfferCBOR
-  commitmentProof Proof
+  segment Proof
 }
 
 type OfferCBOR any
-type Proof any
```

Comment:
Why is this being removed? There is no definition for `Proof` now.

Reply:
I forgot to change proof in a couple of places. Proof was any, but now it will be a CID